commons/c2cgeoportal_commons/models/__init__.py

```diff
@@ -34,6 +34,14 @@
 import sqlalchemy.ext.declarative
 import sqlalchemy.ext.declarative.api
 
+try:
+    from pyramid.i18n import TranslationStringFactory
+    _ = TranslationStringFactory('c2cgeoportal_admin')
+except ImportError as e:
+    def null_translation(s: str) -> str:
+        return s
+    _ = null_translation
```

Comment:
Can't just be:
```
def _(s: str) -> str:
    """ For localized string search ```
    return s
```
Because `TranslationStringFactory` don't return a string, than the translations should be in the views or renderer ...

Reply:
Message extractor fails on _(s: str), note that it search for something like _('ma chaine').

Reply:
OK, I understand, but my first message was unclear because the mine part is that for me we don't needs the try part the content of the expect part should be enough, not?

Reply:
I've taken it from main.py and static.py. We do not need the try if we do not use the model outside of pyramid. Ok, to remove it.

Reply:
+1 for me :-)